README.md

```diff
@@ -63,6 +63,45 @@ Requests to cross-scheme subresources will no longer send `SameSite=Lax` or `Sam
 For example, if http://site.example embeds an image from https://site.example, `SameSite=Lax` and `SameSite=Strict` cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
 ## Questions
+### How does this affect iframes?
+#### The src attribute
+A request to get the content specified by the `src` attribute is treated as any other subresource request and is affected the same ways.
+Once the content is loaded it runs as cross-site or same-site depending on if the `src` value was cross-scheme or not, respectively.
+
+For example, the following request for a page will not be sent `SameSite=Lax` nor `SameSite=Strict` cookies (nor will its response be able to set those cookies) as it's cross-scheme. After the page is loaded nothing on the page is able to access those `SameSite` cookies because it's being run as cross-site.
+```
+<html>
+<body>
+
+<h1>This page is hosted at https://website.example/secure.html</h1>
+<iframe src="http://website.example/insecure.html" title="An iframe to an insecure page"></iframe>
+
+</body>
+</html>
+```
+#### The srcodc attribute
+Content that is loaded into an iframe through the `srcdoc` attribute inherits its parent's origin. Therefore it has the same "Schemeful Same-Site" restrictions (or lack thereof) as its parent document.
+
+### How does this affect scripts?
+Scripts run as part of their parent document, therefore they have the same "Schemeful Same-Site" restrictions (or lack thereof) as their parent document.
```

Comment:
The other order might work better. That is, there are two independent things here, and this order is the reverse of how they actually happen:

1. What cookies does *fetching* the script send? This is based on usual subresource rules.
2. What happens to `document.cookie` accesses made while *executing* the script? This is based on whether the embedding document could access the relevant cookies.

Talking about them in that order lets you emphasize in (2) that it is independent from (1). (Although anyone pulling in a cross-origin script ought to already know this, since this is just how cross-origin scripts work in general. I worry that we keep making this seem more complicated than it is. :-) There are reasonably general principles that most of this falls out of.)

Reply:
I reordered the headers.

I'd agree with you but I'm already seeing some confusion with this. Most everything in this explainer comes from the first principle of "different scheme == different site" and all the current (cookie) restrictions that entails. However, internalizing this fact seems to be difficult which leads to the many questions.

I'm hoping it's helpful to continue reinforcing that principle with repeated examples.